ref/Interaction_with_Geometry.md

```diff
@@ -185,6 +188,15 @@ You can find the valid parameters for elements by constructing them manually and
 
 ------
 
+#### Creating a geometric element with no given name: `create(‹string›,‹list2›)`
+
+Behaves as the 3-arity `create`, but the list of element names as first arguments is not given and the names of the elements are chosen by CindyJS.
+If an algorithm (such as `"IntersectionCircleCircle"`) gives multiple geometric objects then a list of the created elements is returned instead of a single element. However, if a modifier `pos` is specified, then only the point closest to `pos` is created and returned.
+
+If too many arguments are given in `‹list2›`, then those arguments are ignored and the last given argument will be interpreted as a modifier `pos`, if `pos` has not been defined yet.
```

Comment:
Let us discuss this during the next meeting (@richter-gebert currently wants to use create$2 with this usage). I will introduce some output as warning when the slicing is happening to make debugging easier.

Reply:
Actually I don’t understand why we need a 2-ary version at all. Can anybody enlighten me?

Reply:
We thought about the primitives to write an editor for the geometric straight-line-program within CindyScript. It turned out that the CindyScript code for several operations drastically blows up if the user has to specify the names of the output objects (for instance, if one wants to assign reasonable names, one has to know the type of the result of an algorithm in beforehand).
Furthermore, we are working on a randomized prover, which will detect the creation of duplicate elements (for instance a line that connects two points that have already been connected or more complicated incidences through Pappos theorem). In this case, it should not be possible to define a second GeoObject having another name and both create$2 and create$3 should not create a new element but return the existing one. In this case, it a create$3 might be confusing for the user, because the specified name might be different from the name of the created element (In this case, it might be also a reasonable convention to trigger a rename)

Reply:
As far as I recall the rename actually happens in Cinderella. 

So, we should add this to Cinderella as well, I guess.